include/CFL/PAGGrammar.txt

```diff
@@ -1,22 +1,11 @@
 Start:  
    V
 Productions:
-   V -> Gepbari V Gepi;
-   F -> N1;
-   N1 -> epsilon;
-   N1 -> N1 Copy;
-   F -> N2;
-   N2 -> epsilon;
-   N2 -> N2 Store V Load;
-   Fbar -> N4;
-   N4 -> epsilon;
-   N4 -> N4 Copybar;
-   Fbar -> N5;
-   N5 -> epsilon;
-   N5 -> N5 Loadbar V Storebar;
+   F -> ( Copy ) *;
+   F -> ( Store V Load ) *;
+   Fbar -> ( Copybar ) *;
+   Fbar -> ( Loadbar V Storebar ) *;
    V -> Fbar V F;
+   V -> Gepbari V Gepi;
```

Comment:
how do you distinguish an attribute sym from a normal sym?

Is `Gepbari` a normal sym? If not, we will need to use a special form for example `Gepbar_i`

Reply:
Gepbari will be a attribute symble
since 'i' is at the end of the Gepbar,
so in this case Gepi will also be the attribute symble.

But I think you suggestion is better than existed one, i will change that.